data/language/dutch.txt

```diff
@@ -2382,19 +2382,19 @@ STR_2380    :{SMALLFONT}{BLACK}Kleiner stuk water aanpassen
 STR_2381    :{SMALLFONT}{BLACK}Groter stuk water aanpassen
 STR_2382    :Land
 STR_2383    :Water
-STR_2384    :{WINDOW_COLOUR_2}Je doel:
+STR_2384    :{WINDOW_COLOUR_2}Je doelstelling:
```

Comment:
http://www.goeievraag.nl/overig/vraag/15264/verschil-tussen-doel-doelstelling-leveren

Een doelstelling is dus een concrete versie van een doel. Als het doel "verkeer veiliger maken" is, is een doelstelling bijvoorbeeld "maximale snelheid van 50km/u". In dit geval is doelstelling dus beter.